.github/workflows/weekly-preview.yml

```diff
@@ -0,0 +1,38 @@
+name: weekly-preview
+
+on:
+  schedule:
+  - cron: "0 2 * * 6"  # 02:00 of every Saturday
+
+jobs:
+  packaging:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version:  3.8
+    - name: Install setuptools
+      run: |
+        python -m pip install --user --upgrade setuptools wheel
+    - name: Build distribution
+      run: |
+        sed -i 's/name\ =\ monai$/name\ =\ monai-weekly/g' setup.cfg
+        git diff setup.cfg
+        # build tar.gz and wheel
+        git config user.name "CI Builder"
+        git config user.name "monai.miccai2019@gmail.com"
+        git add setup.cfg setup.py
+        git commit -m "Weekly build"
+        git tag 0.4.dev$(date +'%y%U')
+        python setup.py sdist bdist_wheel
```

Comment:
is it possible to include the commit hash such as `python -c 'import monai; print(monai.__version__)' `, so that we can identify the source code of a weekly release easily for debugging purposes...

Reply:
Yes.  The information monai.__version__ is controlled by versioneer.  I added 'on-the-fly' commit id of source code to __commit_id__ so users can print(monai.__commit_id__) to show the source code commit which was used to build the PyPi package.